lighthouse-core/scripts/compare-timings.js

```diff
@@ -19,26 +20,43 @@ const ROOT_OUTPUT_DIR = `${LH_ROOT}/timings-data`;
 const argv = yargs
   .help('help')
   .describe({
+    // common flags
     'name': 'Unique identifier, makes the folder for storing LHRs. Not a path',
+    'report-exclude': 'Regex of properties to exclude. Set to none to disable default',
     // --collect
     'collect': 'Saves LHRs to disk',
     'lh-flags': 'Lighthouse flags',
     'urls': 'Urls to run',
     'n': 'Number of times to run',
     // --summarize
     'summarize': 'Prints statistics report',
-    'measure-filter': 'Regex filter of measures to report. Optional',
+    'measure-filter': 'Regex of measures to include. Optional',
     'output': 'table, json',
+    // --compare
+    'compare': 'Compare two sets of LHRs',
+    'delta-sort': 'Property to sort by its delta',
+    'desc': 'Set to override default ascending sort',
```

Comment:
wdyt about `sort-direction` defaults to `desc`? :)

~~why would you care about the timings that are *most* similar?~~

ah it's not absolute delta it's the signed delta? wdyt about incorporating absolute value delta?

Reply:
was trying to avoid long flags tbh